templates/projects/view.tmpl

```diff
@@ -80,35 +80,35 @@
 								{{svg "octicon-kebab-horizontal"}}
 							</div>
 							<div class="menu user-menu" tabindex="-1">
-								<a class="item show-modal button" data-modal="#edit-project-column-modal-{{.ID}}">
+								<div class="item show-modal button" data-modal="#edit-project-column-modal-{{.ID}}">
```

Comment:
This breaks the dropdown's a11y.

![image](https://github.com/go-gitea/gitea/assets/2114189/157417bf-d86a-4424-b69b-e690d1b91872)

Reply:
Interesting, in this case it's `button` > `a` > `div`. Only problem is Fomantic doc examples show `div`, so people might still introduce them.

Reply:
Nope, the `buttion` doesn't fit "dropdown menu". `<a>` is the best choice here.

Reply:
Yeah, just thought to mention it and probably something to watch out for from new PRs. Check HTML of first example:

https://fomantic-ui.com/modules/dropdown.html#dropdown